python/pyarrow/tests/test_feather.py

```diff
@@ -128,19 +128,22 @@ def test_dataset(version):
     num_values = (100, 100)
     num_files = 5
     paths = [random_path() for i in range(num_files)]
-    df = pd.DataFrame(np.random.randn(*num_values),
-                      columns=['col_' + str(i)
-                               for i in range(num_values[1])])
+    table = pa.Table.from_arrays(
```

Comment:
In principle we could also use `pa.table(..` here instead (a bit more ergonomic to use)

Reply:
will switch to that